gufe/settings/models.py

```diff
@@ -42,6 +43,9 @@ class Config:
         arbitrary_types_allowed = False
         smart_union = True
 
+    def _ipython_display_(self, d2d=None):
```

Comment:
why the `d2d` option?

```suggestion
    def _ipython_display_(self):
```

Reply:
just copied this https://github.com/OpenFreeEnergy/gufe/blob/main/gufe/mapping/ligandatommapping.py#L109